modules/silauth/src/Auth/Source/yii

```diff
@@ -8,8 +8,8 @@
  * @license http://www.yiiframework.com/license/
  */
 
-define('YII_DEBUG', false);
-define('YII_ENV', 'production');
+const YII_DEBUG = false;
+const YII_ENV = 'production';
```

Comment:
The define function is better, because if it is already defined, it doesn't change it.

Reply:
See above.